tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_browser_navigation.py

```diff
@@ -10,52 +9,31 @@
 
 class TestBrowserNavigation(GaiaImageCompareTestCase):
 
-    _community_link_locator = (By.CSS_SELECTOR, '#community a')
-    _community_history_section_locator = (By.ID, 'history')
-
     def setUp(self):
         GaiaImageCompareTestCase.setUp(self)
         self.connect_to_local_area_network()
 
-        self.test_url = self.marionette.absolute_url('mozilla.html')
+        self.test_url = self.marionette.absolute_url('Mozhome.html')
 
     def test_browser_navigation(self):
         search = Search(self.marionette)
         search.launch()
         browser = search.go_to_url(self.test_url)
-
         browser.switch_to_content()
-        self.verify_home_page()
 
-        Wait(self.marionette).until(lambda m: m.title == 'Mozilla')
+        Wait(self.marionette).until(lambda m: 'building a better Internet' in m.title)
         self.take_screenshot()
-        community_link = self.marionette.find_element(*self._community_link_locator)
-        # TODO: remove the explicit scroll once bug 833370 is fixed
-        self.marionette.execute_script("arguments[0].scrollIntoView(false);", [community_link])
-        community_link.tap()
 
-        self.verify_community_page()
+        self.marionette.switch_to_frame()
+        self.scroll(browser._browser_frame_locator, 'up', 400)
```

Comment:
Ok, got the explanation from njpark from this. This function is just weird 'up' means here it's scrolling down.